src/components/Timer/Timer.tsx

```
@@ -0,0 +1,55 @@
+
+      <div className="border-primary-0 typography-digital from-primary-0/0 to-primary-0/20 mx-48 flex h-[298px] w-[264px] flex-col items-center justify-center rounded-lg border-2 bg-linear-to-br text-[154px]">
+        {formattedMinutes}
+        <span className="typography-body-small-s">MINUTES</span>
+      </div>
+      <svg
+        width="24"
+        height="112"
+        viewBox="0 0 24 112"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)